build/proj/CoreBuild.proj

```diff
@@ -62,11 +59,10 @@
 
   <Target Name="Build">
     <ItemGroup>
-      <SolutionBuildTarget Include="Rebuild" Condition="'$(Rebuild)' == 'true'" />
-      <SolutionBuildTarget Include="Build" Condition="'$(Build)' == 'true' and '$(Rebuild)' != 'true'" />
+      <SolutionBuildTarget Include="Rebuild;Pack" Condition="'$(Rebuild)' == 'true'" />
+      <SolutionBuildTarget Include="Build;Pack" Condition="'$(Build)' == 'true' and '$(Rebuild)' != 'true'" />
```

Comment:
Why not just change the condition on the Pack SolutionBuildTarget to refer to `BuildOrRebuild`? Is there some other reason why the ordering change is advantageous?

Reply:
The property didn't exist when I wrote that. I've updated it to use it.